### DISCUSSION TITLE
Managed identities to connect to Azure DevOps instead of PAT

### DISCUSSION THREAD

**Discussion by marnheus** [tfsaggregator/aggregator-cli#282] [Ideas]
The usage of managed identities to log into Azure DevOps is now available in public preview.
What would be great would be to enable system assigned managed identity on the Azure Function app hosting the aggregator and using it to log into Azure DevOps.

That would eliminate the need to manage the PAT and it would make it much more clear in ADO who touched the history using an actual system identity.

Here's the doc : https://learn.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/service-principal-managed-identity?view=azure-devops

**Comment by giuliov** [comment]
Need to analyse the changes and the effort.
